resources/migrations/000_migrations.yaml

```diff
@@ -12993,6 +12993,54 @@ databaseChangeLog:
             tableName: metabase_database
             columnName: details
 
+  - changeSet:
+      id: v45.00-044
+      author: metamben
+      comment: Added 0.45.0 -- create app permission graph revision table
+      changes:
+        - createTable:
+            tableName: app_permission_graph_revision
```

Comment:
I'm starting to think renaming the general permissions revision table to `application_permissions_revision` in #21709 was a bad idea... having both `application_permissions_revision` and `app_permission_graph_revision` is a little on the confusing side. I think this one makes sense but we should probably go change the other one at some point. 

I'm also wondering if maybe it would make sense to try to consolidate our various permissions revision tables ... seems weird that we have Collection `namespace` but then we still have to introduce a new table to be able to use it. Not something I think needs to be addressed now tho.

Reply:
My intention was to keep the fact that we're reusing collection permissions an implementation detail. That is, ideally it should not be visible in the API when manipulating permissions and should not be visible in the audit log either.